Cargo.toml

```diff
@@ -5,3 +5,6 @@ members = [
     "cosmos-sdk-proto",
     "cosmrs"
 ]
+
+[patch.crates-io]
+tendermint-proto = { git = "https://github.com/informalsystems/tendermint-rs" }
```

Comment:
This required a few upstream changes to `tendermint-proto` which should be released doon